rules/integrations/aws/privilege_escalation_sts_temp_creds_via_assume_role.toml

```diff
@@ -35,18 +81,26 @@ tags = [
     "Data Source: AWS",
     "Data Source: Amazon Web Services",
     "Data Source: AWS STS",
+    "Resources: Investigation Guide",
     "Use Case: Identity and Access Audit",
     "Tactic: Privilege Escalation",
 ]
 timestamp_override = "event.ingested"
 type = "new_terms"
 
 query = '''
-event.dataset:aws.cloudtrail
-    and event.provider:sts.amazonaws.com
-    and event.action:AssumeRole*
-    and event.outcome:success
-    and user.id:*
+event.dataset: "aws.cloudtrail"
+    and event.provider: "sts.amazonaws.com"
+    and event.action: "AssumeRole"
+    and event.outcome: "success"
+    and not (
+      aws.cloudtrail.user_identity.invoked_by: (
+         "config.amazonaws.com" or
+          "securityhub.amazonaws.com" or
+          "sso.amazonaws.com"
+        ) or
+      aws.cloudtrail.resources.arn: (*Amazon* or *AWS* or *Elastic* or *Wiz* or *DataDog*)
```

Comment:
should these type of exclusions be left to the user? We are not sure of everyone's naming conventions for roles and this may cause gaps that people are not expecting

Reply:
Thanks @imays11 - After our conversation, I agree that we should limit exclusions on assumed ARN naming and let the customer make that decision.